mmcv/ops/rans.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from torch import Tensor
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext('_ext', [
+    'rans_encode_with_indexes', 'rans_decode_with_indexes',
+    'pmf_to_quantized_cdf'
+])
+
+
+def rans_encode_with_indexes(symbols: Tensor,
```

Comment:
Please add docstring and wrap `rans_decode_with_indexes` and `pmf_to_quantized_cdf` into functions.

Reply:
I added the docstrings and wrap functions for these functions.
```suggestion
// Copyright(c) OpenMMLab.All rights reserved.
#include <pybind11/pybind11.h>

#include "pytorch_cpp_helper.hpp"
#include "pytorch_device_registry.hpp"

namespace py = pybind11;
std::string rans_encode_with_indexes_impl(
    const Tensor symbols, const Tensor indexes, const Tensor cdfs,
    const Tensor cdfs_sizes, const Tensor offsets, int num_threads) {
  return DISPATCH_DEVICE_IMPL(rans_encode_with_indexes_impl, symbols, indexes,
                              cdfs, cdfs_sizes, offsets, num_threads);
}

py::bytes rans_encode_with_indexes(const Tensor symbols, const Tensor indexes,
                                   const Tensor cdfs, const Tensor cdfs_sizes,
                                   const Tensor offsets, int num_threads) {
  return rans_encode_with_indexes_impl(symbols, indexes, cdfs, cdfs_sizes,
                                       offsets, num_threads);
}

Tensor rans_decode_with_indexes_impl(const std::string& encoded,
                                     const Tensor indexes, const Tensor cdfs,
                                     const Tensor cdfs_sizes,
                                     const Tensor offsets) {
  return DISPATCH_DEVICE_IMPL(rans_decode_with_indexes_impl, encoded, indexes,
                              cdfs, cdfs_sizes, offsets);
}

Tensor rans_decode_with_indexes(const std::string& encoded,
                                const Tensor indexes, const Tensor cdfs,
                                const Tensor cdfs_sizes, const Tensor offsets) {
  return rans_decode_with_indexes_impl(encoded, indexes, cdfs, cdfs_sizes,
                                       offsets);
}

Tensor pmf_to_quantized_cdf_impl(const Tensor pmfs, const Tensor pmf_lengths,
                                 const Tensor tail_masses) {
  return DISPATCH_DEVICE_IMPL(pmf_to_quantized_cdf_impl, pmfs, pmf_lengths,
                              tail_masses);
}

Tensor pmf_to_quantized_cdf(const Tensor pmfs, const Tensor pmf_lengths,
                            const Tensor tail_masses) {
  return pmf_to_quantized_cdf_impl(pmfs, pmf_lengths, tail_masses);
}

```